code/api/src/modules/user/resolvers.js

```diff
@@ -69,10 +69,13 @@ export async function getAll() {
 }
 
 // Update
-export async function update(parentValue, { id, email }) {
+export async function update(parentValue, { id, email, description }) {
   const user = await models.User.findOne({ where: { email } })
-  if (!user) {
-    return await models.User.update({ email }, { where: { id } })
+  if (!user || user.id === id ) {
```

Comment:
I like the way you wrote this in one line, I did nested iteration and it doesnt look as clean.